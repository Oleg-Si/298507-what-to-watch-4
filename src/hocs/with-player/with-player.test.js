import React from 'react';
import renderer from 'react-test-renderer';
import withPlayer from './with-player.jsx';

const Component = () => {
  return (
    <div>
      <p>Component</p>
    </div>
  );
};

const WrappedComponent = withPlayer(Component);

it(`Проверяет снепшот хока withPlayer`, () => {
  const tree = renderer.create(
      <WrappedComponent
        isMuted={true}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
