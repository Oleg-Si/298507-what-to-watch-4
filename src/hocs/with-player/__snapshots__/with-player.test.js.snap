// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Проверяет снепшот хока withPlayer 1`] = `
<div>
  <p>
    Component
  </p>
</div>
`;
